src/WhoopsErrorHandler.php

```diff
@@ -76,9 +76,18 @@ public function __construct(
      */
     protected function handleException($exception, Request $request, Response $response)
     {
-        $this->prepareWhoopsHandler($request);
+        // Push the whoops handler if any
+        if (null !== $this->whoopsHandler) {
```

Comment:
Can you reverse the order here for the comparison for consistency? I haven't seen the Yoda condition style being used elsewhere in the project.